portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -27,6 +27,6 @@
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
     response.setContentType("text/html;");
-    response.getWriter().println("<h1>Hello world!</h1>");
```

Comment:
Its okay to remove the `<h1>`, but you don't need to if you don't want to! The [h1](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/Heading_Elements) tag  signifies that the text is a "header" text.